api/osrd_infra/views/pathfinding.py

```diff
@@ -174,9 +174,9 @@ def compute_vmax(payload: PathPayload, track_map: Mapping[str, TrackSection]):
     for track_id, track in track_map.items():
         trees[track_id].addi(0, track["length"], -1)
         for speed_section in track["speed_sections"]:
-            assert speed_section["begin"] < speed_section["end"]
-            trees[track_id].chop(speed_section["begin"], speed_section["end"])
-            trees[track_id].addi(speed_section["begin"], speed_section["end"], speed_section["speed"])
+            if speed_section["begin"] > speed_section["end"]:
+                trees[track_id].chop(speed_section["begin"], speed_section["end"])
+                trees[track_id].addi(speed_section["begin"], speed_section["end"], speed_section["speed"])
```

Comment:
this change makes no sense to me

Reply:
There are some speed sections with a length of 0, the assert sometimes failed when using the fuzzer on the new infra. Ideally it shouldn't be imported in the first place, but I didn't want to re-import a new infra just for this

Reply:
the condition seem to be broken, the > sign only runs the conditions where begin is after end, that's just weird

Reply:
Ah good catch

Reply:
:fishing_pole_and_fish: